<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="AddCourse">
     <attribute name="title">
      <string>Add Course</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>181</width>
        <height>541</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>179</width>
         <height>539</height>
        </rect>
       </property>
       <widget class="QPushButton" name="AddCourseButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Course</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="CourseTitle">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="CourseTitleLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Course Title</string>
        </property>
       </widget>
       <widget class="QLabel" name="CourseTime">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="CourseTimeStart">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>118</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>56</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>56</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>End</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="CourseTimeEnd">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>118</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>56</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="CourseLocation">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>290</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>56</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Days</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="CheckBoxMWF">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>350</y>
          <width>88</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>M, W, F</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CheckBoxTTH">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>390</y>
          <width>88</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>T, TH</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="AddAssignment">
     <attribute name="title">
      <string>Add Assignment</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>56</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="AssignmentTitle">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>113</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>56</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Course</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CoursesComboBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>56</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Due Date</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="CourseDateTimeEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Percent Complete</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="AssignmentPercentComplete">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>60</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>56</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddAssignmentButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Assignment</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Friday</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ThursdayCourses">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>101</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="FridayCourses">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>101</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>-10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Notifications</string>
    </property>
   </widget>
   <widget class="QListWidget" name="MondayCourses">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>91</width>
      <height>501</height>
     </rect>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="WednesdayCourses">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>101</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Thursday</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Monday</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </widget>
   <widget class="QListWidget" name="TuesdayCourses">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>91</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="NotificationList">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
